recipes/sentry-native/all/conanfile.py

```diff
@@ -35,7 +36,7 @@ class SentryNativeConan(ConanFile):
         "qt": False,
         "with_crashpad": "sentry",
         "with_breakpad": "sentry",
-
+        "performance": True,
```

Comment:
I will change this to false but I wanted to check if the CI was doing well with this option on.

Reply:
Done here: 42ef07d